### ISSUE TITLE
Plover's machine thread occasionally crashes while reading TX Bolt

### ISSUE THREAD

**Issue by morinted** [openstenoproject/plover#717]
Reproducibility: Rarely

## Summary

When using the *Stenomod* I occasionally run into this exception:

```
  File "/Users/ted/git/plover/plover/machine/txbolt.py", line 89, in run
    STENO_KEY_CHART[(key_set * 6) + i])
IndexError: tuple index out of range
```

It's random and intermittent, I've probably had it some twenty times and I can't get a pattern out of it.

I've been meaning to debug exactly what the machine sends to determine what it's thinking, but thought I should make this issue before I forgot.

**Comment by morinted** [comment]
I never had this happen with the Infinity Ergonomic, so it very well could be a fault in the Stenomod firmware that I'm using.

**Comment by morinted** [comment]
Fixed in #721
